terraform/workspace-variables/review.tfvars.json

```diff
@@ -25,5 +25,7 @@
   "deploy_azure_backing_services": true,
   "enable_monitoring": false,
   "enable_postgres_ssl": true,
-  "add_database_name_suffix": true
-}
+  "add_database_name_suffix": true,
+  "aks_worker_app_instances" : 2
```

Comment:
It's only 1 in paas: https://github.com/DFE-Digital/teaching-vacancies/blob/main/terraform/workspace-variables/review.tfvars.json#L19
The  line can be removed as 1 is default